ts/features/payments/transaction/components/PaymentsLegacyAttachmentBottomSheet.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { View } from "react-native";
+import { Body, H4, VSpacer } from "@pagopa/io-app-design-system";
+import {
+  IOBottomSheetModal,
+  useIOBottomSheetAutoresizableModal
+} from "../../../../utils/hooks/bottomSheet";
+import I18n from "../../../../i18n";
+
+/**
+ * This custom hook, usePaymentsLegacyAttachmentBottomSheet, is designed to display a bottom sheet
+ * containing detailed information about where the user can find the legacy transaction recepit.
+ */
+const usePaymentsLegacyAttachmentBottomSheet = (): IOBottomSheetModal => {
+  const getModalContent = () => (
+    <View>
+      <ParagraphWithTitle
+        title={I18n.t(
+          "features.payments.transactions.legacy.bottomSheet.noticeHeading"
+        )}
+        body={I18n.t(
+          "features.payments.transactions.legacy.bottomSheet.noticeBody"
+        )}
+      />
+      <VSpacer size={24} />
+      <ParagraphWithTitle
+        title={I18n.t(
+          "features.payments.transactions.legacy.bottomSheet.allNoticesHeading"
+        )}
+        body={I18n.t(
+          "features.payments.transactions.legacy.bottomSheet.allNoticesBody"
+        )}
+      />
+      <VSpacer size={48} />
+    </View>
+  );
+
+  const modal = useIOBottomSheetAutoresizableModal({
+    component: getModalContent(),
+    title: (
+      <H4>
+        {I18n.t("features.payments.transactions.legacy.bottomSheet.title")}
+      </H4>
+    )
+  });
+
+  return { ...modal };
+};
```

Comment:
We could simplify this hook as follows.
The `useIOBottomSheet` hook internally uses the `H4` typography component to render the title.
And we can return directly the composed hook, avoiding to use of the spread operator.
What do you think?

```suggestion
/**
 * This custom hook, usePaymentsLegacyAttachmentBottomSheet, is designed to display a bottom sheet
 * containing detailed information about where the user can find the legacy transaction recepit.
 */
const usePaymentsLegacyAttachmentBottomSheet = (): IOBottomSheetModal =>
  useIOBottomSheetAutoresizableModal({
    title: I18n.t("features.payments.transactions.legacy.bottomSheet.title"),
    component: (
      <View>
        <ParagraphWithTitle
          title={I18n.t(
            "features.payments.transactions.legacy.bottomSheet.noticeHeading"
          )}
          body={I18n.t(
            "features.payments.transactions.legacy.bottomSheet.noticeBody"
          )}
        />
        <VSpacer size={24} />
        <ParagraphWithTitle
          title={I18n.t(
            "features.payments.transactions.legacy.bottomSheet.allNoticesHeading"
          )}
          body={I18n.t(
            "features.payments.transactions.legacy.bottomSheet.allNoticesBody"
          )}
        />
        <VSpacer size={48} />
      </View>
    )
  });
```